build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -82,8 +80,7 @@ export class FluidPackageCheck {
 		const testScript = testMochaScript ?? pkg.getScript(testScriptName);
 		if (testScript && /(ts-)?mocha/.test(testScript)) {
 			const shouldHaveConfig =
-				pkg.monoRepo?.kind === MonoRepoKind.Client ||
-				pkg.monoRepo?.kind === MonoRepoKind.Azure;
+				pkg.monoRepo?.kind === "client" || pkg.monoRepo?.kind === "azure";
```

Comment:
nit: should there be constants for the known values? Or not worth it?

Reply:
`fluidPackageCheck.ts` will be going away or port to repoPolicyCheck.   I don't want to bother making this nice.